packages/toast/src/toast.css

```diff
@@ -20,10 +20,12 @@ governing permissions and limitations under the License.
 
     opacity: 0;
     pointer-events: none;
-    transition: transform var(--spectrum-overlay-animation-duration) ease-in-out,
+    transition:
+        transform var(--spectrum-overlay-animation-duration) ease-in-out,
         opacity var(--spectrum-overlay-animation-duration) ease-in-out,
         visibility 0s linear var(--spectrum-overlay-animation-duration);
     visibility: hidden;
+    overflow-wrap: anywhere; /* Allow long texts to wrap in the toast */
```

Comment:
Shouldn't this be happening on the CSS side?

Reply:
Yes! I tested in CSS and issue lies there too. 
CSS PR: https://github.com/adobe/spectrum-css/pull/3073